### DISCUSSION TITLE
Project admin not active

### DISCUSSION THREAD

**Discussion by Shelbybosss** [sanjay-kv/Open-source-Practice#885] [Q&A] [answered]
![flag](https://github.com/GSSoC24/Open-source-Practice/assets/102911609/963c8344-c356-4fab-a6cf-e5c0bbcdf828)
 Its been almost a day no response from admin side.

**Comment by PRAYAG0908** [comment, marked as answer]
added that repository url in https://github.com/GSSoC24/Contributor/discussions/94 gssoc will flag that admin

**Comment by aditi75432** [comment]
Wait for some more time. The admin might have a lot of workload. Even if admin does not respond, flag the project

**Comment by sanjay-kv** [comment]
Thanks this is set to remove
